### ISSUE TITLE
require issue with StructField/StructType

### ISSUE THREAD

**Issue by doronrosenberg** [EclairJS/eclairjs-nashorn#200]
I had to add

```
var StructField = require(EclairJS_Globals.NAMESPACE + '/sql/types/StructField');
```

to `StructType.js`

and 

```
var DataType = require(EclairJS_Globals.NAMESPACE + '/sql/types/DataType');
```

to `StructField.js`

The code that was causing this:

```
var jsc = new SparkContext("local[*]", "sql.DataFrameReader Integration Tests");

var SQLContext = require(EclairJS_Globals.NAMESPACE + '/sql/SQLContext');
var sqlcontext1 = new SQLContext(jsc);
var dataFrameReader1 = sqlcontext1.read();
var dataFrame1 = dataFrameReader1.jdbc("jdbc:mysql://localhost:3306/test", "test", {"user":"root","password":"mypass"});

JSON.stringify(dataFrame1.collect())
```

**Comment by jtbarbetta** [comment]
Looks like we just missed wrapping the sql/types package properly.  Fixing that now.  Sorry.

**Comment by jtbarbetta** [comment]
Fixed with commit 64c14cd1e13bf219c35e7b1f7b35c774a6c5671e.
